src/ILVerification/src/Verifier.cs

```diff
@@ -140,6 +174,32 @@ public IEnumerable<VerificationResult> Verify(PEReader peReader, MethodDefinitio
             }
         }
 
+        void TmpLogToRemove(string str)
+        {
+            Console.WriteLine(str);
+        }
+
+        private IEnumerable<VerificationResult> VerifyInterface(EcmaModule module, TypeDefinitionHandle typeDefinitionHandle)
+        {
+            EcmaType type = (EcmaType)module.GetObject(typeDefinitionHandle);
+            
+            TmpLogToRemove(type.Name);
+
+            var builder = new ArrayBuilder<VerificationResult>();            
+
+            foreach (DefType interfaceDef in type.ExplicitlyImplementedInterfaces)
+            {                           
+                foreach (MethodDesc interfaceMethodDesc in interfaceDef.GetAllMethods())
+                {
+                    // MethodDesc mimpl = type.ResolveInterfaceMethodTarget(interfaceMethodDesc);
+                    // MethodDesc mimpl = type.ResolveInterfaceMethodToVirtualMethodOnType(interfaceMethodDesc);
+                    var mi = MetadataVirtualMethodAlgorithm.ResolveVariantInterfaceMethodToVirtualMethodOnType(interfaceMethodDesc, type);
```

Comment:
I found some methods to check if class implements interface. But this mehods checks only for "virtual". Do you think it's a good idea add some overloads and expose a method to match also non virtual method?Or it's better rewrite code on verifier?

Reply:
I mean page 157 of ECMA-35 states "Interfaces can define virtual methods, but only if those methods are abstract" 
If i compile with Vs i get abstract virtual methods...but that "can" should states that i can have a method on interface "non virtual", right?

Reply:
The standard allows interfaces to have static methods.

Reply:
thank's @MichalStrehovsky!It's better rewrite new alg on verifier(taking a cue from codebase)...